sdk/textanalytics/Azure.AI.TextAnalytics/src/StringIndexType.cs

```diff
@@ -3,10 +3,10 @@
 
 using Azure.Core;
 
-namespace Azure.AI.TextAnalytics.Models
+namespace Azure.AI.TextAnalytics
 {
     [CodeGenModel("StringIndexType")]
-    internal partial struct StringIndexType
+    public partial struct StringIndexType
```

Comment:
why are we making it public?

Reply:
StringIndexType is part of EntitiesTaskParameters.cs and other task parameters class which causes accessibility errors if this is not public

Reply:
We agreed on not exposing StringIndexType in our SDK so better hide it again